libs/design-system/src/atoms/inputs/selectInputs.tsx

```diff
@@ -78,10 +78,11 @@ export const SelectInput = ({
       id={id}
       name={id}
       className={cn('antialiased border border-solid rounded-md p-2 focus:outline-none', variations[type], className)}
-      onChange={handleChange}>
-      {options.map(({ id, label, value }) => (
-        <option key={id} id={id} label={label} selected={value === (initialValue || options[0].value)}>
-          {value}
+      onChange={handleChange}
+      defaultValue={initialValue || options[0].value}>
```

Comment:
Nice one! Didn't know that!

Reply:
Me neither but I kept getting an eeror in the console asking to do this 🙈